life_expectancy/cleaning.py

```diff
@@ -13,32 +13,23 @@ def save_data(df): # pylint: disable=invalid-name
     filepath = Path('./life_expectancy/data/pt_life_expectancy.csv')
     df.to_csv(filepath, index = False)
 
-def convert_int_float(df):  # pylint: disable=invalid-name
-    """Convert value to float and year to int"""
-    df['value'] = df['value'].str.extractall(r'(\d+.\d+)').astype(float).unstack().max(axis=1)
-    df['value'] = pd.to_numeric(df['value'], errors='coerce')
-    df['year'] = df['variable'].astype(int)
-    return df
-
-def drop_columns_rows(df):  # pylint: disable=invalid-name
-    """Drop uneeded columns and rows"""    
-    df = df.dropna(axis = 0)
-    return df[['unit', 'sex', 'age', 'region', 'year', 'value']]
-
-def to_pivot():
-    """Receive data and convert it to pivot"""  
-    raw_data = get_data()
-
+def clean_data(raw_data, reg):  # pylint: disable=missing-function-docstring
```

Comment:
**suggestion**: Instead of using the pylint disable comment at every functions, declare the exception in the pyproject.toml or .pylintrc file